### ISSUE TITLE
Search episodes by keyword

### ISSUE THREAD

**Issue by tonytamsf** [Podcastindex-org/search#3]
Are there plans to create an api to search the episodes (not tied to a feed) by keyword?

A use case would be to find episodes about a specific topic or interviews with a specific author and I want to listen to all the recent interviews across all feeds.

I'm a contributor on AntennaPod.

**Comment by daveajones** [comment]
Yep.  That'll be a thing for sure.  Have to fry some bigger fish first, but that'll be done.   If you want to help with anything join the podcastindex.social mastodon instance.  It's a good place to talk about that stuff.

I'll leave this issue open for now until some initial code goes up.
